package.json

```diff
@@ -23,7 +23,7 @@
     "eslint": "eslint . --ignore-path .prettierignore",
     "format": "npm run eslint -- --fix && npm run prettier -- --write && sort-package-json",
     "test": "npm run eslint && npm run prettier -- --list-different && npm run build-production",
-    "prettier": "prettier \"**/*.{js,md,css}\"",
+    "prettier": "prettier \"**/*.{js,md,css}\" --write",
```

Comment:
Why this change? This breaks the scripts. `format` script is for writing changes, `test` script is for the CI, and should fail if the files aren't correct (shouldn't write).

Reply:
sorry, I did not mean to push this